client/verta/verta/operations/monitoring/labels.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+
+
+from verta._protos.public.monitoring.Summary_pb2 import FilterQuerySummarySample
+
+from verta._protos.public.monitoring.Labels_pb2 import (
+    FindSampleLabelsRequest,
+    FindSampleLabelValuesItem,
+    FindSampleLabelValuesRequest,
+)
+from .summaries import Summary
+from . import time_utils
+from .utils import maybe
+
+class Labels:
+    def __init__(self, conn, conf):
+        self._conn = conn
+        self._conf = conf
+
+
+    def find_keys(self, **kwargs):
+        """
+            Find a list of labels according to provided parameters.
+
+            :key summary_query:
+            :key sample_ids:
+            :key labels:
+            :key time_window_start:
+            :key time_window_end:
+            :return: find_keys should return a list of strings used as label keys
+            :rtype: list
```

Comment:
I'd prefer these to be NumPy-style docstrings, but open to discuss/change some other time.